### ISSUE TITLE
Consider adjusting wording on Add Additional Payments Methods page

### ISSUE THREAD

**Issue by dmallory42** [Automattic/woocommerce-payments#7040]
### Description

We have a task which is shown to the user when they enable UPE (note, this would be rarely seen because UPE is on by default, and the method of disabling it is fairly well hidden/hard to find).

I wonder if we can make the wording more clear here. I am not sure if `Enablement complete` is very easily understood - I wonder if something like `Additional payment method setup complete` or `Setup complete` is clearer? This could be just me, so I decided to open an issue for consideration.

cc. @elizaan36 for thoughts 🙂

<img width="696" alt="Screenshot 2023-08-22 at 11 02 25" src="https://github.com/Automattic/woocommerce-payments/assets/11770181/317a62c8-8372-4bcb-9ea0-210089c79421">

Other changes to make to this page:

- Remove the `see Terms and Conditions` text on the credit/debit card pill - it was introduced in [this PR](https://github.com/Automattic/woocommerce-payments/pull/6949) and the solution could be to update `client/utils/account-fees.tsx` to conditionally only add this text if a param is passed in (so we can control where it shows and where it doesn't)
- Remove the `required` label, and instead use a disabled checkbox with a tooltip saying `required`: https://github.com/Automattic/woocommerce-payments/issues/7040#issuecomment-1703121761.

**Comment by elizaan36** [comment]
> or Setup complete is clearer

+1. "Setup complete" is what I would use here. Much more clear.

There's one thing that's new that needs to be tweaked a bit. Could you share where this "see Terms and Conditions" was added to the fees pill so I can see the context? There's also a strange "Required" text. The placement of these texts needs to be thought through a bit better.
<img width="596" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/5121465/78ebd79e-eb04-4316-818e-c5d308820bf4">

**Comment by dmallory42** [comment]
Hi @elizaan36!

That `Terms and conditions` text looks like an unintended change introduced by [this PR](https://github.com/Automattic/woocommerce-payments/pull/6949) - I can create a new issue to handle removing it from this fees pill if you think it should not be there?

I'm not sure where that `Required` text comes from, but I agree it could be improved.

**Comment by elizaan36** [comment]
Yes the "see Terms and Conditions" link and the "required" text can be removed. I think the checkbox could be read only with a tooltip that says "Required".

**Comment by elizaan36** [comment]
I shared an example of a disabled checkbox in this comment `paJDYF-9Mc#comment-20578`. We don't really have this state built into the component though.

**Comment by dmallory42** [comment]
Thanks @elizaan36! I updated this issue with those changes as well. 🙂

**Comment by zmaglica** [comment]
This issue impacts Additional Payment methods, so assigning to team Fusion (based on team responsibilities Pc2DNy-3z-p2) @anu-rock .
Assigning as part of Gamma Triage process PcreKM-yM-p2.

**Comment by elizaan36** [comment]
This issue has to do with the presentation of fees for users who have the incentive applied. cc @rogermattic for awareness. As we update the fees display will be good to keep in mind what information merchants need to see regarding the incentive. 

@dmallory42 In an earlier comment we discussed removing the "Terms and conditions" link, but now I'm wondering is it legally required? cc @vladolaru

**Comment by vladolaru** [comment]
@elizaan36 As I answered here (pbUcTB-447-p2#comment-9272), the Terms and conditions link is not required on this page _as long as_ we have it the Payment > Overview > Account details section.

**Comment by frosso** [comment]
Closing - the "Add additional payment methods" wizard this issue is about no longer exists. The whole task was removed in #10730.

The T&C link on the fees pill was already dropped separately in #8458, and the "required" checkbox is gone along with the rest of the wizard. So all three points here are moot now.
